native-schema-registry/src/main/java/com/amazonaws/services/schemaregistry/ExceptionWriter.java

```diff
@@ -30,4 +34,22 @@ private static void writeException(String message, C_GlueSchemaRegistryErrorPoin
         throwError(errorPointer, errorMessageAsCString.get(), FAILURE_CODE);
         errorMessageAsCString.close();
     }
+
+    private static String getFullStackTraceMessage(Throwable throwable){
+        if (throwable == null || throwable.getMessage() == null) {
+            return "NULL is thrown";
+        }
+        if (throwable.getStackTrace() == null){
```

Comment:
Can we still set the error message in this case?

Reply:
I have updated the logic here.